pkg/docker/docker.go

```diff
@@ -542,6 +666,9 @@ func (d *stiDocker) GetScriptsURL(image string) (string, error) {
 	if err != nil {
 		return "", err
 	}
+	if imageMetadata == nil {
```

Comment:
under what condition is imagemetadata nil, but we didn't get an error on CheckAndPullImage?